apis_core/generic/templates/generic/generic_confirm_delete.html

```diff
@@ -0,0 +1,20 @@
+{% extends basetemplate|default:"base.html" %}
+
+{% block content %}
+  <div class="modal-dialog">
+    <div class="modal-content">
+      <div class="modal-body">
+        <form action="" method="post">
+          {% csrf_token %}
+          <h4>
+            Confirm deletion of:
+            <div class="text-center">
+              <strong>{{ object }}</strong>
+            </div>
```

Comment:
`<div>` tags should not be placed inside HTML headings. An inline tag which can be used in this place is `<span>`.

If the `<div>` is needed for formatting purposes, it needs to go around the `<h4>` tag.